crates/services/p2p/src/config.rs

```diff
@@ -254,6 +258,7 @@ impl Config<NotInitialized> {
             tx_pool_threads: 0,
             state: NotInitialized,
             subscribe_to_pre_confirmations: true,
+            subscribe_to_transactions: false,
```

Comment:
We want to enable it by defautl for tests

```suggestion
            subscribe_to_transactions: true,
```